presets/cloudflare/firewall_events/preset.yaml

```diff
@@ -0,0 +1,362 @@
+name: cloudflare_firewall_events
+title: "Cloudflare Firewall Events"
+author: Antimatter
+description: "Cloudflare Firewall Events logs"
+iconURL: "https://raw.githubusercontent.com/antimatterhq/dasl-content-packs/refs/heads/main/presets/cloudflare/firewall_events/icon.png"
```

Comment:
Should we use `dasl-v1.0` instead of `main` here? (and in other presets?)

Although it's best to host them on some CDN